scripts/jenkins-support.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+function connect_rally_server_run_test
+{
+    # Connect to rally server and run tests
+    ssh -T root@$rally_server "bash -s $cloud $image_name" <<'EOF'
+cloud=$1
+image_name=$2
+
+source .openrc
+# Check if jeos image already exist on cloud
+openstack image show -c name --format value $image_name
+image_exist=$?
+# Upload jeos image
+if [[ $image_exist != 0 ]]; then
```

Comment:
This can be more cleanly expressed as:

if ! openstack image show -c name --format value $image_name; then